docs/source/get_started/modular_design.rst

```diff
@@ -28,6 +28,12 @@ This class can contain learnable parameters and `NaN` (missing value) handling.
 It takes :class:`~torch_frame.data.TensorFrame` as input and applies stype-specific feature encoder (specified via :obj:`stype_encoder_dict`) to :obj:`~torch.Tensor` of each stype to get embeddings for each :obj:`~torch_frame.stype`.
 The embeddings of different :obj:`stypes<torch_frame.stype>` are then concatenated to give the final 3-dimensional :obj:`~torch.Tensor` :obj:`x` of shape :obj:`[batch_size, num_cols, channels]`.
 
+.. note::
+    Different :obj:`stypes<torch_frame.stype>` can have the same internal representation, e.g. both :class:`~torch_frame.embedding` and :class:`~torch_frame.text_embedded` are stored with :class:`~torch_frame.data.MultiEmbeddingTensor`.
+    In PyTorch Frame, all :obj:`stypes<torch_frame.stype>` sharing the same data structure have child-parent dependency.
+    The child :obj:`stype<torch_frame.stype>` will be unified to the parent :obj:`stype<torch_frame.stype>` after materialization.
+    We consider the :class:`~torch_frame.embedding` as the parent of :class:`~torch_frame.text_embedded` and only parent :obj:`stype<torch_frame.stype>` is supported in the :obj:`stype_encoder_dict`.
```

Comment:
Can we expand and improve this description?

Child stype is user-facing stype. Parent stype specifies how data is grouped in tensor frame. Tell more about motivation.